express/scripts/scripts.js

```diff
@@ -822,15 +823,19 @@ export function decorateBlock(block) {
     let shortBlockName = blockName;
     block.classList.add('block');
     // begin CCX custom block option class handling
-    if (shortBlockName !== 'how-to-steps-carousel') {
-      blocksWithOptions.forEach((b) => {
-        if (shortBlockName.startsWith(`${b}-`)) {
-          const options = shortBlockName.substring(b.length + 1).split('-').filter((opt) => !!opt);
-          shortBlockName = b;
-          block.classList.add(b);
-          block.classList.add(...options);
-        }
-      });
+    for (let i = 0; i < blocksWithOptions.length; i += 1) {
+      const b = blocksWithOptions[i];
```

Comment:
Why change from
```js
blocksWithOptions.forEach((b) => {
```
to a `for` loop?

Reply:
to be able to `break` it. 
As  'how-to-steps' is a substring of 'how-to-steps-carousel', inside loop test would apply too, we need to exist the loop.